pkg/api/api_test.go

```diff
@@ -242,52 +243,48 @@ func TestPostageHeaderError(t *testing.T) {
 	var (
 		mockStorer     = mock.NewStorer()
 		mockStatestore = statestore.NewStateStore()
-		logger         = logging.New(ioutil.Discard, 0)
+		logger         = logging.New(ioutil.Discard, 5)
 		client, _, _   = newTestServer(t, testServerOptions{
 			Storer: mockStorer,
 			Tags:   tags.NewTags(mockStatestore, logger),
 			Logger: logger,
 		})
-	)
 
-	for _, tc := range []struct {
-		name     string
-		endpoint string
-		batchId  []byte
-		expErr   bool
-	}{
-		{
-			name:     "bytes",
-			endpoint: "/bytes",
-			batchId:  []byte{0},
-			expErr:   true,
-		},
-		{
-			name:     "bytes",
-			endpoint: "/bytes",
-			batchId:  []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}, //32 bytes - ok
-		},
-		{
-			name:     "bytes",
-			endpoint: "/bytes",
-			batchId:  []byte{}, // empty batch id falls back to the default zero batch id, so expect 200 OK. This will change once we do not fall back to a default value
-		},
-		{
-			name:     "bzz",
-			endpoint: "/bzz",
-			batchId:  []byte{0},
-			expErr:   true,
-		},
-	} {
+		batchInvalid = []byte{0}
+		batchOk      = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0} //32 bytes - ok
```

Comment:
r: if you want to short your byte slice literal you can define it as `[]byte{0: 0, 31: 0}`